test/e2e/framework/metrics_util.go

```diff
@@ -0,0 +1,856 @@
+/*
```

Comment:
bad rebase? not sure where this file came from

Reply:
Bad rebase, they refactored under me